aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -333,4 +333,10 @@ static inline int64_t computeLRWorkDim(const char jobz, int64_t m, int64_t n) {
   return std::max(5 * mn * mn + 5 * mn, 2 * mx * mn + 2 * mn * mn + mn);
 }
 
+static inline void checkUplo(std::string uplo) {
```

Comment:
Add a comment describing this function and its intended use. `const std::string&`?